glambie/processing/processing_helpers.py

```diff
@@ -549,7 +549,8 @@ def set_unneeded_columns_to_nan(data_catalogue: DataCatalogue) -> DataCatalogue:
 
 def get_reduced_catalogue_to_date_window(data_catalogue: DataCatalogue,
                                          start_date: float,
-                                         end_date: float) -> DataCatalogue:
+                                         end_date: float,
+                                         date_window_is_gap: bool = False) -> DataCatalogue:
```

Comment:
might be better to make this variable name a bit more verbose
```suggestion
                                         date_window_is_data_gap_window: bool = False) -> DataCatalogue:
```